src/Sylius/Behat/Context/Api/Admin/BrowsingProductVariantsContext.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use ApiPlatform\Api\IriConverterInterface;
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Behat\Context\Api\Resources;
+use Sylius\Component\Core\Model\ProductInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Webmozart\Assert\Assert;
+
+final class BrowsingProductVariantsContext implements Context
+{
+    public function __construct (
+        private ApiClientInterface $client,
+        private ResponseCheckerInterface $responseChecker,
+        private IriConverterInterface $iriConverter,
+    ) {
+    }
+
+    /**
+     * @When I view all variants of the product :product
+     */
+    public function iViewAllVariantsOfTheProduct(ProductInterface $product): void
+    {
+        $this->client->index(
+            Resources::PRODUCT_VARIANTS,
+            [
+                'product' => $this->iriConverter->getIriFromResource($product),
+                'order[position]' => 'asc',
+            ],
+        );
+    }
+
+    /**
+     * @When I start sorting variants by position
+     */
+    public function iSortProductsByPosition(): void
```

Comment:
```suggestion
    public function iStartSortingVariantsByPosition(): void
```
There is no option to change the sorting order(only `desc`).

Reply:
But I don't see a reason why we should allow it now if we don't need it 🤷🏼‍♂️.